rosetta/services/constants.go

```diff
@@ -0,0 +1,13 @@
+package services
+
+const (
+	RosettaSDKVersion = "1.4.1"
```

Comment:
Not critical in this PR, I will create an issue for this... but it would be better if we could get this value directly from the SDK. Otherwise, it is get out of sync quickly

Related issue: https://github.com/Zondax/rosetta-filecoin-proxy/issues/21